tools/mktutorial/templates/tutorial.tmpl

```diff
@@ -19,7 +19,7 @@ layout: how-to-guide
 <!-- To change it, please see https://github.com/pulumi/docs/tree/master/tools/mktutorial. -->
 
 <p class="mb-4 flex">
-    <a class="flex flex-wrap items-center rounded text-xs text-white bg-blue-600 border-2 border-blue-600 px-2 mr-2 whitespace-no-wrap hover:text-white" style="height: 32px" href="{{GitHubURL}}" target="_blank">
+    <a class="flex flex-wrap items-center rounded-md text-lg text-white bg-blue-600 border-2 border-blue-600 px-2 mr-2 whitespace-no-wrap hover:text-white" style="height: 45px; font-family: 'Gilroy'; " href="{{GitHubURL}}" target="_blank">
```

Comment:
this is the thing to review, everything else is generated

Reply:
I think for this, you can use the `font-display` class as it’s defined as Gilroy in the Tailwind config.

Reply:
ah cool, didnt know